### ISSUE TITLE
Conflict with RBCN fertilizer

### ISSUE THREAD

**Issue by beckyeeky** [nt153133/Retainers#1]
“栽培模式” is actually fertilizer botbase in RBCN.
And thus they have conflict.
You can use retainers no matter what locale.
We get it.
Any way it does not work with StB FFXIV.

**Comment by nt153133** [comment]
Yeah when I made it I used the fertilizer bot base as a starting point. Also I'm not sure if the select string will work correctly in other languages. Can you tell what the what the Chinese characters for 'retainers' Is?

**Comment by beckyeeky** [comment]
“雇员”

**Comment by nt153133** [comment]
I changed the name but I'm thinking it might have an issue since I refer to all the retainer selection menu by their English lines.

**Comment by beckyeeky** [comment]
Yes I tried to look for regex in the code but couldn’t find.
I am sure it is because I am noob in coding.
In CN StB server, it is still one string that involves all info such as
name, venture, Selling status.
Now in EN ShB it is cut into a few sections, name job Gil etc.

Kindly tricky.
-- 
Beck --------------------------- 一朝一夕，夢若昏曉。 發自 iOS 行動版 Gmail

**Comment by nt153133** [comment]
Is it at least selecting a retainer? if it is and getting stuck at the next screen then you'd need to change the select strings at the bottom of retainerTasks.cs But also the code literally searches for "Summoning Bell" in helperFuntions.cs so i think it's screwed from the start. It allows for it to use any bell but does kinda limit the language options.
